apps/microtvm/zephyr/template_project/microtvm_api_server.py

```diff
@@ -424,6 +432,39 @@ def _get_platform_version(self, zephyr_base: str) -> float:
 
         return float(f"{version_major}.{version_minor}")
 
+    def _load_cmsis(self, lib_path: Union[str, pathlib.Path], cmsis_path: Union[str, pathlib.Path]):
+        """Copy CMSIS header files to generated project."""
+
+        cmsis_path = pathlib.Path(cmsis_path)
+
+        lib_path = pathlib.Path(lib_path)
+        if not lib_path.exists():
+            lib_path.mkdir()
+
+        include_directories = ["CMSIS/DSP/Include", "CMSIS/DSP/Include/dsp", "CMSIS/NN/Include"]
```

Comment:
just curious why we copy rather than just add to include path e.g. via `-I` or `target_include_directories` in CMakeLists.txt?

Reply:
Either would work, this approach has less affect on CMakeLists, also easier for debugging a project since all files would be accessible in the generated project. But I can see that this could be a bad solution if two header files have similar name and we copy everything here. Happy to change it if you suggest?

Reply:
yeah the case I was more thinking about was if CMSIS-NN reorganizes and a file inside one of these directories starts depending on a file from another, non-copied directory. I sort of prefer the `-I` for this reason, though it does mean you have to not move CMSIS-NN. On the other hand, examining CMakeLists is a fairly logical thing to do if you get include path problems.

Reply:
cool, I'll change it

Reply:
PTAL! thanks!